### DISCUSSION TITLE
[自定义] 鼠标经过头像，头像不要旋转

### DISCUSSION THREAD

**Discussion by itning** [tangly1024/NotionNext#2046] [Ideas]
**为什么提出这个新的特性改动**
现在鼠标经过头像，头像会自动旋转

**描述一下你推荐的解决方案**
给一个环境变量，配置成false，不允许头像旋转

**描述一下你考虑过的其它替代解决方案**
改源码

**补充说明**
N/A

**Comment by tangly1024** [comment]
通过修改代码禁用头像旋转的示例：

删除 `hover:rotate-45` 这个class即可

https://github.com/tangly1024/NotionNext/blob/ce3f5f4eada523215c17a6416ae8187cbb4c86cf/themes/hexo/components/InfoCard.js#L11
